app/main.py

```diff
@@ -1,3 +1,85 @@
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        if isinstance(other, int | float):
```

Comment:
The use of `int | float` is not valid in Python versions prior to 3.10. Consider using `isinstance(other, (int, float))` for compatibility with earlier versions.